ext/datasources/helium/helium.c

```diff
@@ -1644,6 +1627,30 @@ helium_cursor_update(WT_CURSOR *wtcursor)
 }
 
 /*
+ * helium_cursor_reserve --
+ *	WT_CURSOR.reserve method.
+ */
+static int
+helium_cursor_reserve(WT_CURSOR *wtcursor)
+{
+	CURSOR *cursor;
+
+	cursor = (CURSOR *)wtcursor;
+
+#ifdef XXX_BROKEN_KEITH
+	/*
+	 * XXX
+	 * We don't currently support reserve, this will require some work.
+	 * The test programs don't currently detect it, so return success.
+	 */
+	return (0);
+#else
+	(void)cursor;
+	return (0);
```

Comment:
No, because `format` doesn't have any way to avoid `reserve` operations. (If you want, I could add code to `format` to make `reserve` operations configurable.)